spinegeneric/cli/generate_figure.py

```diff
@@ -1317,7 +1312,7 @@ def main(argv=sys.argv[1:]):
 
         # Excluded sites
         logger.info(
-            "Sites removed: {}".format(list(df[df["exclude"] == True]["site"].values))
+            "Sites removed: {}".format(list(df[df["exclude"]]["site"].values))
```

Comment:
Ditto: this was a vectorized operation. Is it still?

Reply:
Ditto: reinstated the `== True` comparison and added a `noqa: 712` comment.